webapp/src/main/webapp/news-details/components/ExoNewsFavoriteAction.vue

```diff
@@ -2,6 +2,7 @@
   <favorite-button
     :id="activityId"
     :favorite="isFavorite"
+    :hidden=true
```

Comment:
```suggestion
    hidden
```
You can just add the name of the attribute to change its value to `true`
See https://v2.vuejs.org/v2/guide/components-props.html#Passing-a-Boolean